rviz_default_plugins/src/rviz_default_plugins/displays/map/palette_builder.cpp

```diff
@@ -124,5 +124,17 @@ std::vector<unsigned char> makeRawPalette()
   return palette_builder->buildPalette();
 }
 
+std::vector<unsigned char> makeProbabilityPalette()
+{
+  auto palette_builder = std::make_shared<PaletteBuilder>();
+  // Standard gray map palette values
+  for (int i = 0; i < 255; i++) {
```

Comment:
Considering that the [occupancy grid message](https://github.com/ros2/common_interfaces/blob/d4505b57ed20c49853350705ea72eb6f89b2eafb/nav_msgs/msg/OccupancyGrid.msg#L9) documents values are in the range [0, 100], doesn't it make sense to iterate up to 100?
If so, then I believe the math works out to be identical to the "map" pallet, which makes sense the the values of an occupancy grid are intended to be probabilities.

Reply:
See above with the `makeRawPalette()`, there's already infrastructure for raw costmaps.

(might be abusing it, but that's how it goes)

Reply:
The raw and costmap palettes are much different. I guess the question is why is the map palette not enough? It is essentially the same as the map palette except that it assumes a different scale ([0, 100] vs [0, 254]). You can achieve the same grey colors by scaling the input map values before they reach RViz. Alternatively we could add a property to select the scale, then it would work for any map assuming a different value for P = 1.

Reply:
Certainly not trying to argue that this is the most useful thing in the world, but its another way to represent the raw probability information. I view this as a case as "better is good". 

For your comment on [0,100] vs [0, 254], I personally can't argue with you much. That seems to make sense to me. However this was written by Clearpath and adopted by Rover so I'm inclined to go with "someone has a good reason". However if it turns out no one has a good reason, more than happy to change to 0-100 as I agree that makes more sense (if people are using the message how they should be, which... yeah).

@rotu what are your thoughts?